service/src/main/resources/config/aou/test/SC2023Q3R1/sql/t_condition_occurrence_all.sql

```diff
@@ -12,3 +12,9 @@ ON p.person_id = co.person_id
 
 LEFT JOIN `all-of-us-ehr-dev.SC2023Q3R1.visit_occurrence` AS vo
 ON vo.visit_occurrence_id = co.visit_occurrence_id
+
+LEFT JOIN `all-of-us-ehr-dev.SC2023Q3R1.concept` AS c
```

Comment:
nit: Can this just be a `JOIN`? Since you're filtering on the `domain_id` below, I think it's required to be non-null. ditto for the other SQL files.

Reply:
will do and merge.

Reply:
done